src/platform/firefox/manifest.json

```diff
@@ -87,6 +87,7 @@
         "menus",
         "storage",
         "notifications",
+        "clipboardRead",
```

Comment:
Can we make this an optional permission?

Some users might not like that the extension can read their clipboard.

Reply:
This is now an optional permission. The permission will be requested when a user calls the function `readText()` (currently only the background script calls this) or if the clipboard feature will be enabled in the settings. 

If the extension has no permission to read the clipboard but clipboard autoclean was enabled in the settings, it will overwrite the clipboard without knowing what is in it. If the extension can read the clipboard it will only overwrite it when the content matches the password.